### ISSUE TITLE
Packaging Issue

### ISSUE THREAD

**Issue by Balaji-Ramseshkumar** [syncfusion/angular2-seeds#26]
This application has been packaged successfully. When opening .exe file it displays the below error, kindly advice on this☺

![package_error 2](https://user-images.githubusercontent.com/20086829/30542073-86b3e76c-9c9b-11e7-8d02-dce2f08cc64d.PNG)

**Comment by karthickthangasamy** [comment]
@Jilaba Thanks for your interest in Syncfusion Products.

Could you please share the below information, which help us to provide you prompt solution at the earliest.?
* System environment (NPM and Node versions)
* How you are converting Angular web application into .exe application.?
    * Are you developing application using `Electron` for desktop environment.?
* Information about your application scope/setup.
* If possible, share your application in GitHub repo.

Thanks,
Karthick T

**Comment by AbinayaSubbiah** [comment]
Thanks for your information about the System environment and sample. 

We made the below changes to pack the Angular application into Electron app. 

-  Created electron app for running the application in development mode. (src/electron)
 

-  In index.html local file references are moved to package import like the below to avoid failures.

      src/deps/default.css
      src/deps/images

- Implemented the below scripts to launch the electron app for development mode and create packagers.

npm run build-electron | Build the Angular application and copy the required files into dist folder
-- | --
npm run electron | Rans the command npm run build-electron and launch the electron app for the generated folder dist.
npm run package | Rans the command npm run build-electron and pack the Angular application into electron app using electron-packager

For your convenience, we attached the modified sample in the below link. 

[sample](http://www.syncfusion.com/downloads/support/directtrac/general/ze/ej-angular-electron1786494339.zip) 

Extract the sample and run the below commands to launch and package the application. 

```javascript
npm install 
npm run electron              // it will launches the application locally 
npm run package                // it will generate application package 
```
